doc/languages-frameworks/vim.section.md

```diff
@@ -203,6 +203,8 @@ Note: this is not possible anymore for Neovim.
 
 Nix expressions for Vim plugins are stored in [pkgs/applications/editors/vim/plugins](https://github.com/NixOS/nixpkgs/tree/master/pkgs/applications/editors/vim/plugins). For the vast majority of plugins, Nix expressions are automatically generated by running [`./update.py`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/editors/vim/plugins/update.py). This creates a [generated.nix](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/editors/vim/plugins/generated.nix) file based on the plugins listed in [vim-plugin-names](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/editors/vim/plugins/vim-plugin-names). Plugins are listed in alphabetical order in `vim-plugin-names` using the format `[github username]/[repository]@[gitref]`. For example https://github.com/scrooloose/nerdtree becomes `scrooloose/nerdtree`.
 
+After running `./update.py`, if nvim-treesitter received an update, also run [`nvim-treesitter/update.py`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/editors/vim/plugins/update.py) to update the tree sitter grammars for `nvim-treesitter`.
```

Comment:
This should be done programmatically otherwise I will bet money that it will be forgotten.

Reply:
Do you think we should just add this to the vim update script? it takes a relatively long time to run and it would be wasted if they were going to revert the `vimPlugins: update` commit to e.g. not cause merge conflicts when adding new plugins

Reply:
How hard is it to determine if it is necessary to run? If not very much then I think we should do it.

When are we reverting entire vimPlugin updates? We should always only revert specific plugins.

Reply:
the vim update script doesnt support adding new plugins without updating all the existing plugins iirc. by revert i just meant deleting the commit to avoid merge conflicts